realm/realm-library/src/main/java/io/realm/internal/CheckedRow.java

```diff
@@ -133,7 +133,7 @@ public OsMap getRealmAnyMap(long columnIndex) {
     @Override
     public OsMap getModelMap(long columnIndex) {
         RealmFieldType fieldType = getTable().getColumnType(columnIndex);
-        if (fieldType != RealmFieldType.STRING_TO_MIXED_MAP) {
+        if (fieldType != RealmFieldType.STRING_TO_LINK_MAP) {
```

Comment:
Was this a bug? 😮

Reply:
Looks like, it wasn't hit til we used it on dynamic realms..